src/app/dangermessage/dangermessage.component.ts

```diff
@@ -0,0 +1,29 @@
+import { Component, OnInit } from '@angular/core';
+class danger{
```

Comment:
there is no need for this class definition here unless you were just practicing classes